designer/client/src/components/graph/node-modal/Variable.tsx

```diff
@@ -68,7 +68,7 @@ export default function Variable({
                 {renderFieldLabel("Variable Name")}
             </LabeledInput>
             <EditableEditor
-                fieldName="expression"
+                fieldName="$expression"
```

Comment:
What is the reason for the change of the param name to the `$` prefix? Is it our new convention? 
As is see, we still have https://github.com/TouK/nussknacker/pull/4953/files#diff-cca035cb72539c4c5ea0a2f36b88c9a47476645295c68c0cb477d34c3720fc41L31 the field name without `$` prefix `$fields-${index}-key` (I mean key which is the field name).

Reply:
Ok, and what about this one `$fields-${index}-key`, shouldn't it be `$fields-${index}-$key`?

Reply:
I think this is the first case besides fragment parameters when were introducing nested fieldNames like this so there's no convention. For me the $ before nested name `key` could make sense if we were mapping these errors and after mapping one field would expect a fieldName `$key` but when we do it flatly like currently, I can't see the point, unless for consistency with other changes? WDYT?

Reply:
Ok, thats reasonable, I'll update for consistency :)